src/azure-cli-core/azure/cli/core/commands/constants.py

```diff
@@ -31,5 +32,6 @@
 
 BLACKLISTED_MODS = ['context', 'shell', 'documentdb', 'component']
 
-SURVEY_PROMPT = Fore.YELLOW + Style.BRIGHT + 'Please let us know how we are doing: ' + Fore.BLUE \
-    + 'https://aka.ms/clihats' + Style.RESET_ALL
+SURVEY_PROMPT = 'Please let us know how we are doing: https://aka.ms/clihats'
+SURVEY_PROMPT_COLOR = Fore.YELLOW + Style.BRIGHT + 'Please let us know how we are doing: ' + Fore.BLUE \
```

Comment:
you can add some snapshot in PR description for this kind of ux change

Reply:
Nice idea!

Originally with color:

![image](https://user-images.githubusercontent.com/4003950/77139963-ec55f280-6ab2-11ea-8985-36225d97654f.png)

After running `$Env:AZURE_CORE_NO_COLOR='true'`. It becomes 

![image](https://user-images.githubusercontent.com/4003950/77139984-fa0b7800-6ab2-11ea-81c2-ea5f55136b74.png)